queries.sql

```diff
@@ -9,4 +9,49 @@ SELECT date_of_birth FROM animals WHERE name IN('Agumon', 'Pikachu');
 SELECT name, escape_attempts FROM animals WHERE weight_kg > 10.5;
 SELECT name, neutered FROM animals WHERE neutered = True;
 SELECT name FROM animals WHERE name != 'Gabumon';
-SELECT * FROM animals WHERE weight_kg >= 10.4 AND weight_kg <= 17.3;
\ No newline at end of file
+SELECT * FROM animals WHERE weight_kg >= 10.4 AND weight_kg <= 17.3;
+
+
+BEGIN;
```

Comment:
- [ ] In the project requirements, there is a point about the queries you have not accounted for. Take a look at the following screenshot 👇 
![image](https://user-images.githubusercontent.com/90222110/199990670-ce7416d7-c27e-4eaa-9571-f6877c64dc61.png)

Please add the query as mentioned above here also 😃 👍